### PR TITLE
add logo

### PR DESCRIPTION
**Pull request by xroll**
# Pull Request Template

## Description

Add logo folder and logo

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by AndreVarandas** [comment]
Hi @xroll thank you for your PR! 

Do you want to add also a link to it, so it is displayed in the README.md file?

**Comment by xroll** [comment]
I make a new one
logo

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-
-<a  href="https://www.freepnglogos.com/images/fortnite-png-27038.html" ><img  src="https://www.freepnglogos.com/uploads/fortnite-png/fortnite-logo-logo-fortnite-png-photo-icons-and-png-6.png"
-width="200" alt="fortnite logo, logo fortnite png photo icons and png" /></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/0Xkf3WD/logo.png" alt="logo" border="0"></a>
```
Comment (AndreVarandas) [inline README.md:1]: @xroll we can use github links directly like this:

`logo`

Would you do this last change? Thank you!

**Review by AndreVarandas — CHANGES_REQUESTED** [review]
Please, have a look into my comment

File: README.md (hunk 1 above)
Comment (xroll) [reply README.md:1]: Yes, understood !

**Comment by xroll** [comment]
Is it good ?

**Comment by AndreVarandas** [comment]
I'ts perfect! Thank you @xroll !

**Pull request merged**